### PR TITLE
 Enrolling Vitess in LFX 2023 Term 3 Sep-Nov projects

### PR DESCRIPTION
**Pull request by frouioui**
Hello @nate-double-u, this PR adds the proposal for the Vitess' LFX project during the next term.

### CODE REVIEW

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Hi @frouioui, I'm a little confused—this already looks to be set up. Are you proposing a second term of this?

Could you add Vitess to the Table of Contents?

**Comment by frouioui** [comment]
> Hi @frouioui, I'm a little confused—this already looks to be set up. Are you proposing a second term of this?
> 
> Could you add Vitess to the Table of Contents?

Hey @nate-double-u. I am proposing a second term to continue what the current intern has started. Should I open a second issue on the `vitessio/arewefastyet` repository or keep the same as the first term?

I'll change the table of content and remove the URL.

File: programs/lfx-mentorship/2023/03-Sep-Nov/README.md [hunk 1]
```diff
@@ -556,6 +556,20 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+#### Rework the frontend UI of Vitess’ benchmarking tools
+
+- Description: Vitess uses arewefastyet to automatically benchmark its codebase and ensure no performance regression is introduced. The mentee will have the responsibility of continuing the UI that was previously created using React/Vite.
+- Expected Outcome: The expected outcome is to continue working on the Frontend UI that was developed during the 2023-summer term, that includes adding an admin UI, adding a feature to ensure the consistency of the results, and improving the overall aspect of the website.
+- Recommended Skills: React +++, Docker ++, Vite +, REST API +++, Golang ++
+
+- Mentor(s):
+  - @fouioui Florent Poinsard frouioui@planetscale.com
+- Upstream Issue (URL): https://github.com/vitessio/arewefastyet/issues/328
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/8299d27a-9e36-4de6-abbc-c9282634ee03
```
[the remark this answers is not in the document]
Comment (frouioui) [reply programs/lfx-mentorship/2023/03-Sep-Nov/README.md]: Fixed via c62e776.

**Comment by frouioui** [comment]
> Would there be a new mentee? Unfortunately mentees can only participate in one LFX Mentorship term.

@nate-double-u yes, there would be a new mentee.

**Comment by tanayvaswani** [comment]
@nate-double-u I think @frouioui has some plans with the project that was also a part of previous term, and he wants somebody new as a mentee this time, to be working ahead with him, am I right @frouioui? 
Also, from an applicant's point of view it would be great if you can create a new issue and mention all the progress which has been done by the previous mentee @frouioui. 
Thanks :)

**Comment by frouioui** [comment]
> I think @frouioui has some plans with the project that was also a part of previous term, and he wants somebody new as a mentee this time, to be working ahead with him, am I right @frouioui?

That's totally correct @tanayvaswani, thank you!

> Also, from an applicant's point of view it would be great if you can create a new issue and mention all the progress which has been done by the previous mentee @frouioui.

Good idea, I will do that!

**Comment by frouioui** [comment]
[A new issue](https://github.com/vitessio/arewefastyet/issues/415) has been linked.

**Comment by nate-double-u** [comment]
Thanks for the updates!

Adding the hold label as this needs to be enrolled in the LFX platform and approved by them.
I'll add the LFX URL link once I get it, and will add it as a suggested change.

File: programs/lfx-mentorship/2023/03-Sep-Nov/README.md [hunk 2]
```diff
@@ -556,6 +558,20 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+### Vitess
+
+#### Continue the migration to React and enhance existing frontend UI
+
+- Description: Vitess uses arewefastyet to automatically benchmark its codebase and ensure no performance regression is introduced. The mentee will have the responsibility of continuing the UI that was previously created using React/Vite.
+- Expected Outcome: The expected outcome is to continue working on the Frontend UI that was developed during the 2023-summer term, that includes adding an admin UI, adding a feature to ensure the consistency of the results, improving the overall UX of the website, and add new pages to improve the scope of arewefastyet. The full list of expected work can be found in the issue linked below.
+- Recommended Skills: React +++, Docker ++, Vite +, REST API +++, Golang ++
+
+- Mentor(s):
+  - @fouioui Florent Poinsard frouioui@planetscale.com
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/README.md:570]:
```suggestion
  - Florent Poinsard @fouioui frouioui@planetscale.com
```

File: programs/lfx-mentorship/2023/03-Sep-Nov/README.md [hunk 3]
```diff
@@ -556,6 +558,20 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
...
+- Upstream Issue (URL): https://github.com/vitessio/arewefastyet/issues/415
+- LFX URL:
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2023/03-Sep-Nov/README.md:572]:
```suggestion
- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/10d70edd-60ec-409b-8801-0fb752501b12
```

**Review by nate-double-u — APPROVED** [review]
Please do not merge PR till we get LFX approvals.

**Comment by nate-double-u** [comment]
LFX approved, removing hold.

**Comment by frouioui** [comment]
Thank you @nate-double-u!

**Pull request merged**